recipes/mod_wsgi/yum_requirements.txt

```diff
@@ -0,0 +1,4 @@
+httpd-devel
```

Comment:
I think we would need a CDT of this if we are linking to it. If not, we can drop the `-devel` and just use the `httpd` package.

Reply:
ok

Reply:
APXS is in httpd-devel